### ISSUE TITLE
This is not a real issue

### ISSUE THREAD

**Issue by InFog** [jonathas/top10ips#1]
Hey,

So, it's bad that you didn't get the job. I have being in the same situation one year ago.

Implementing the code in 10 languages shows you are interested in proving your point, and this is good.

Now, try to thing better about these implementations. They kind of a translation from a main implementation. Maybe some of the languages have special points you are missing. It's like people writing Python with a "Java accent", you know what I mean?

And, most of all think about the complexity of your algorithm. Will it scale?
I recommend reading about Big-O notation https://en.wikipedia.org/wiki/Big_O_notation and how to measure the complexity of your software. There are some nice free courses at Coursera and other sites like this on this topic.

Think about why they said no and learn from it. Things like algorithm complexity and scalability are very important at companies like Google. Don't you think so?

Best!

**Comment by jonathas** [comment]
So, you prove my point that this format of interview doesn't prove our real level of overall coding and problem-solving skills. 
Yes, I agree that I might be missing on special points of some of these languages, but I think that's normal, as for example this has been my first contact with Groovy and Lua. So, I've just researched what I needed to in order to develop these examples, and I've never done anything more complex than this with them, yet.

Though I agree that algorithm complexity and scalability are important for them, I believe what they requested at that time was simpler than that.
Well, I believe we should always be improving. I'll take a look at Coursera. Thank you for the tips.
